pipelines/lidar/pipeline.py

```diff
@@ -0,0 +1,138 @@
+import xarray as xr
+import numpy as np
+import cmocean
+import matplotlib.pyplot as plt
+from typing import Dict
+
+from tsdat import IngestPipeline, get_start_date_and_time_str, get_filename
+from utils import add_colorbar, format_time_xticks
+
+
+class Lidar(IngestPipeline):
+    """--------------------------------------------------------------------------------
+    LIDAR INGESTION PIPELINE
+
+    Ingest of Lidar data from an AXYS Technologies buoy stationed off the coast of
+    Humboldt, CA.
+
+    --------------------------------------------------------------------------------"""
+
+    def hook_customize_dataset(self, dataset: xr.Dataset) -> xr.Dataset:
+        # Set initial drive position on wind direction
+        drive_position = None
```

Comment:
Should drive position be included as a global attribute? It seems like this might be useful metadata